commands/utils/archive.js

```diff
@@ -109,6 +109,16 @@ function archive_batch(lt_config, batch) {
         } else if ((!lt_config["run_settings"]["cypress_config_file"])) {
             archive.append("{}", { name: constants.CYPRESS_CONFIG_NAME });
         }
+        if (lt_config["run_settings"]["reporter_config_file"]){
```

Comment:
We should add handling and add default value in set_args

Reply:
Possible cases:
1. If key present but provided file not exists, then the reject condition will handle it 

2. Presence of this key is not mandatory i.e If this key is not present in ltconfig or is empty string, then this if condition will not execute and at next level i.e liisa > lima > lnrc level, it wont be an issue because of the intrinsic property of golang assumes empty value of a key if not present while unmarshalling. 
This key will be provided by user only if they want to generate the report.
@japneetlambdatest plz suggest if any case is missing